### PR TITLE
fix: Corrected grammatical mistakes on the `README.md` file.

### PR DESCRIPTION
**Pull request by gvirus21**
# With the following PR, I have fixed some grammatical mistakes in the `README.md` file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ The documentation is divided into several sections:
 * [Where to Get Support](https://reactjs.org/community/support.html)
 * [Contributing Guide](https://reactjs.org/docs/how-to-contribute.html)
 
-You can improve it by sending pull requests to [this repository](https://github.com/reactjs/reactjs.org).
+You can improve it, by sending pull requests to [this repository](https://github.com/reactjs/reactjs.org).
```
Comment (gvirus21) [inline README.md:36]: Added the required comma after "You can improve it".

File: README.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ You'll notice that we used an HTML-like syntax; [we call it JSX](https://reactjs
 
 ## Contributing
 
-The main purpose of this repository is to continue evolving React core, making it faster and easier to use. Development of React happens in the open on GitHub, and we are grateful to the community for contributing bugfixes and improvements. Read below to learn how you can take part in improving React.
+The main purpose of this repository is to continue evolving the React core, making it faster and easier to use. Development of React happens in the open on GitHub, and we are grateful to the community for contributing bugfixes and improvements. Read below to learn how you can take part in improving React.
```
Comment (gvirus21) [inline README.md:59]: Added required article 'the'

File: README.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 React is a JavaScript library for building user interfaces.
 
-* **Declarative:** React makes it painless to create interactive UIs. Design simple views for each state in your application, and React will efficiently update and render just the right components when your data changes. Declarative views make your code more predictable, simpler to understand, and easier to debug.
+* **Declarative:** React makes it painless to create interactive UIs, design simple views for each state in your application, and React will efficiently update and render just the right components when your data changes. Declarative views make your code more predictable, simpler to understand, and easier to debug.
```
Comment (gvirus21) [inline README.md:5]: Replaced unnecessary `.` with a `,` and corrected the letter-casing.

**Review by gvirus21 — COMMENTED** [review]
Finished reviewing my changes

File: README.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ React has been designed for gradual adoption from the start, and **you can use a
 * [Add React to a Website](https://reactjs.org/docs/add-react-to-a-website.html) as a `<script>` tag in one minute.
 * [Create a New React App](https://reactjs.org/docs/create-a-new-react-app.html) if you're looking for a powerful JavaScript toolchain.
 
-You can use React as a `<script>` tag from a [CDN](https://reactjs.org/docs/cdn-links.html), or as a `react` package on [npm](https://www.npmjs.com/package/react).
+You can use React as a `<script>` tag from a [CDN](https://reactjs.org/docs/cdn-links.html) or as a `react` package on [npm](https://www.npmjs.com/package/react).
```
Comment (prabodhmeshram) [inline README.md:19]: Not sure if we need to remove the comma here, two independent sentence can have comma before the `or`

**Review by AbdullahWins — COMMENTED** [review]
well done!

**Pull request closed without merging**